docs/ui/vercel-ai.md

```diff
@@ -2,6 +2,9 @@
 
 Pydantic AI natively supports the [Vercel AI Data Stream Protocol](https://ai-sdk.dev/docs/ai-sdk-ui/stream-protocol#data-stream-protocol) to receive agent run input from, and stream events to, a [Vercel AI Elements](https://ai-sdk.dev/elements) frontend.
 
+!!! note "AI SDK Version Compatibility"
+    The base protocol is compatible with AI SDK v5 and later. However, [tool approval](#tool-approval) (human-in-the-loop) features require **AI SDK v6 or later**.
```

Comment:
- I don't think AI SDK is relevant here, as on the frontend the user would be using AI Elements, not the SDK. (Note that this could be wrong) Does AI Elements also only support this from a specific version?
- I think we should make this feature opt-in through a flag on `VercelAIAdapter`, both because of the people using older versions of Vercel AI on the frontend, _and_ because they may have already implemented their own `DeferredToolRequests` handling from before this was natively supported in AI Elements / Pydantic AI.